org.nomacs.ImageLounge.json

```diff
@@ -0,0 +1,134 @@
+{
+    "app-id": "org.nomacs.ImageLounge",
+    "branch": "3.14",
```

Comment:
`"default-branch": "stable"`

Reply:
I get this warning:
```
flatpak-builder-WARNING **: 13:30:05.686: Unknown property default-branch for type BuilderManifest
```
if I replace "branch" with "default-branch".

Reply:
Update your flatpak-builder version?

Reply:
Debian used to ship newer flatpak-builder in backports but that depends on your distro of course. You can also install `org.flatpak.flatpak-builder` from Flathub. It has some limitations but it's usually good enough.